keps/sig-cli/20201216-kubectl-default-container.md

```diff
@@ -99,13 +100,14 @@ Issues https://github.com/kubernetes/kubernetes/issues/96986 opened by @howardjo
 
 ## Proposal
 
+A single and generic annotation for all above commands like `kubectl.kubernetes.io/default-container` is a good choice to avoid needing many new annotations in the future.
 
+
+Dropped proposal:
```

Comment:
I believe you can remove the initial proposal from the docs.

Reply:
👌